devfile.yaml

```diff
@@ -14,7 +14,8 @@ commands:
       label: "JBang - Trust Apache organization"
       component: tools
       workingDir: ${PROJECTS_ROOT}/kaoto-examples
-      commandLine: "jbang trust add https://github.com/apache/"
+      # Taken from where the JBang cli is installed in the universal developer image
+      commandLine: "/home/tooling/.sdkman/candidates/jbang/current/bin/jbang trust add -o --fresh --quiet https://github.com/apache/"
```

Comment:
have you tested the image?

When doing similar things in previous attempts, it was breaking the right access to several folders and I wasn't able anymore to change the default jbang version for instance

Reply:
tested and it seems to work now with this repository